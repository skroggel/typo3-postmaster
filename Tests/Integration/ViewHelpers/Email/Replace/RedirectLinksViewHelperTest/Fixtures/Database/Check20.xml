<?xml version="1.0" encoding="utf-8"?>
<dataset>

    <tx_postmaster_domain_model_queuemail>
        <uid>1</uid>
        <pid>1</pid>
        <from_name>Test</from_name>
        <from_address />
        <reply_to_name>Reply Name</reply_to_name>
        <reply_to_address>reply@testen.de</reply_to_address>
        <return_path>return@testen.de</return_path>
        <subject>Test</subject>
        <body_text />
        <attachment />
        <campaign_parameter />
        <plaintext_template />
        <html_template />
        <calendar_template />
        <priority>0</priority>
        <sorting>0</sorting>
        <status>1</status>
        <type>0</type>
        <attachment_type />
        <attachment_name />
        <layout_paths />
        <partial_paths />
        <template_paths />
        <pipeline>0</pipeline>
        <settings_pid>0</settings_pid>
        <category />
    </tx_postmaster_domain_model_queuemail>

</dataset>
