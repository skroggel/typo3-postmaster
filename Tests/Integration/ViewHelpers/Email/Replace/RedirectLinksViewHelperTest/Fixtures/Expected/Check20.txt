


    Testen Sie hier: http://www.example.com/umleitungsseite-der-umleitungen/postmaster/redirect/1?no_cache=1&tx_postmaster_tracking%5Burl%5D=http%3A%2F%2Fwww.google.de
    Testen Sie da: [http://www.example.com/umleitungsseite-der-umleitungen/postmaster/redirect/1?no_cache=1&tx_postmaster_tracking%5Burl%5D=http%3A%2F%2Fwww.yahoo.de%2F%23anchor-1
    Testen Sie dort: mail@example.com
    Testen Sie überall: #anchor-2



    <p>
        <a href="http://www.example.com/umleitungsseite-der-umleitungen/postmaster/redirect/1?no_cache=1&tx_postmaster_tracking%5Burl%5D=http%3A%2F%2Fwww.google.de">Testen Sie hier</a>
        <a href="http://www.example.com/umleitungsseite-der-umleitungen/postmaster/redirect/1?no_cache=1&tx_postmaster_tracking%5Burl%5D=http%3A%2F%2Fwww.yahoo.de%2F%23anchor-1">Testen Sie da</a>
        <a href="mailto:mail@example.com">Testen Sie dort</a>
        <a href="#anchor-2">Testen Sie überall</a>
    </p>



